prow/cmd/marque/main.go

```diff
@@ -106,12 +111,15 @@ func replaceSecret(c *kube.Client) error {
 	}
 
 	s := kube.Secret{
```

Comment:
might be worth setting the type to [`kubernetes.io/tls`](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L3831) too